src/brevitas/proxy/groupwise_int_parameter_quant.py

```diff
@@ -14,6 +14,11 @@ def group_dim(self):
     def group_size(self):
         return self.quant_injector.group_size
 
+    def apply_input_view(self, x):
```

Comment:
Worth making a GroupwiseQuantProxyMixin which provides these functions? Looks like that a lot is shared between `GroupwiseWeightQuantProxyFromInjector` & `GroupwiseActQuantProxyFromInjector`...

Reply:
After further review, I seem to recall there was some issue with Proxy Mixins (perhaps with dependency injection) results in the ExportMixin needing to be in a certain location. I'll leave this for now - we can revisit at another time.